Os/Stub/test/CMakeLists.txt

```diff
@@ -10,6 +10,10 @@ if (NOT BUILD_TESTING)
 endif ()
 add_custom_target("${FPRIME_CURRENT_MODULE}")
 
+add_compile_options(
```

Comment:
This will affect a bunch of os implementations.  We probably want to fix errors there.

Reply:
Captured in #3366.